### DISCUSSION TITLE
Error message on ibaxc transfer

### DISCUSSION THREAD

**Discussion by ewetdans** [IBAX-io/go-ibax#50] [Bug Submission]
Runtime error
×
Early transaction time
![ibax h2](https://user-images.githubusercontent.com/96885317/149627739-ecdcd7c9-e0fe-404f-8eb2-4b0f1e82ec24.png)

![ibax h1](https://user-images.githubusercontent.com/96885317/149627785-8087786a-e5b0-4e1b-bd10-95ddcef6e74e.png)

**Comment by HalloKyle** [comment]
what's question man？
